index.js

```diff
@@ -43,12 +43,13 @@ if (!YARNHOOK_BYPASS) {
     console.log("CMD:", CMD);
   }
 
-  if (lockfile !== null) {
+  if (lockfile) {
```

Comment:
Is there a reason you changed this? If I'm not wrong it can only be "npm", "yarn", or `null`.

Reply:
I figured it would be better to check if it's truthy. I can change it back if you want.

Reply:
I'd like to keep the PR only to relevant stuff, if you don't mind. 😊